components/l10n/common/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+  "+absl/types/optional.h",
```

Comment:
this doesn't seem right. the proper way is to use `third_party/abseil-cpp/absl/types/optional.h`

Reply:
More recent code in Chromium used the above. I spoke with @iefremov and @bridiver on previous PRs and there was no preference.

Reply:
> More recent code in Chromium used the above

this is widely used in `third_party` libraries, but I haven't found this anywhere in `chrome`, `base`, `components`, etc. I see that you've used it already a lot in other parts of brave-ads, so that's fine then.

Reply:
I will raise a PR to change all places back to third_party, thanks for checking :-)